src/Components/Inquiry/Views/InquiryArtistsInCollection.tsx

```diff
@@ -9,22 +9,32 @@ import {
 import { Box, Stack, Text } from "@artsy/palette"
 import { CollectorProfileArtistsAdd } from "Components/CollectorProfile/CollectorProfileArtistsAdd"
 import { useInquiryContext } from "Components/Inquiry/Hooks/useInquiryContext"
+import { useUpdateMyUserProfile } from "Components/Inquiry/Hooks/useUpdateMyUserProfile"
 import { FC } from "react"
+import { Environment } from "react-relay"
 import { useTracking } from "react-tracking"
 import { useAnalyticsContext } from "System/Hooks/useAnalyticsContext"
 import { useOnce } from "Utils/Hooks/useOnce"
 
 export const InquiryArtistsInCollection: FC = () => {
   const { next, relayEnvironment, context } = useInquiryContext()
 
+  const { submitUpdateMyUserProfile } = useUpdateMyUserProfile({
+    relayEnvironment: relayEnvironment.current as Environment,
+  })
+
   const {
     contextPageOwnerId,
     contextPageOwnerSlug,
     contextPageOwnerType,
   } = useAnalyticsContext()
   const { trackEvent } = useTracking()
 
-  useOnce(() => {
+  useOnce(async () => {
+    await submitUpdateMyUserProfile({
+      promptedForUpdate: true,
+    })
+
     const payload: EditProfileModalViewed = {
```

Comment:
@leodmz we're currently firing a "EditProfileModalViewed" event when the user is shown the "add artists to collection" modal. Should we differentiate between the "complete profile" and "add artists..." prompt, or is it OK to use the same event for both impressions?

cc: @dzucconi

Reply:
My idea was to use the same event for both impressions to save me the trouble of having to create a separate event and later stitch them together to get all impressions